### DISCUSSION TITLE
cpp/basics/mutable

### DISCUSSION THREAD

**Discussion by giscus** [imarvinle/CSGuide#74] [Announcements]
# cpp/basics/mutable

编程指北: 计算机系统学习指南, 操作系统, 计算机网络, C++, Java, 算法, 数据结构

https://csguide.cn/cpp/basics/mutable.html

**Comment by Viiic** [comment]
“如果需要在const函数里面修改一些跟类状态无关的数据成员，那么这个函数就应该被mutable来修饰，并且放在函数后后面关键字位置。”

这里应该是这些数据成员应该用mutable来修饰吧？
